CONTRIBUTING.md

```diff
@@ -94,14 +94,15 @@ download and install anything:
 * On each documentation page, there should be an "Improve This Page" link at the very
   top.
 * Click on that link to open the respective source file (usually an `.rst` file in the
-  `doc` folder) on GitHub for editing online (you'll need a GitHub account).
+  `doc` folder or a `.py` file in the `examples` folder) on GitHub for editing online
+  (you'll need a GitHub account).
 * Make your desired changes.
 * When you're done, scroll to the bottom of the page.
 * Fill out the two fields under "Commit changes": the first is a short title describing
   your fixes; the second is a more detailed description of the changes. Try to be as
   detailed as possible and describe *why* you changed something.
-* Click on the "Commit changes" button to open a
-  [pull request (see below)](#pull-requests).
+* Choose "Create a new branch for this commit and start a pull request." and
```

Comment:
```suggestion
* Choose "Create a new branch for this commit and start a pull request" and
```

I know this GitHub option includes a period but I think that the bullet list sentence looks better without the period in the quote and the (partial) quote is still accurate without it